packages/matrix-identity-server/src/config.json

```diff
@@ -10,7 +10,7 @@
   "database_ssl": false,
   "database_user": "",
   "database_vacuum_delay": 3600,
-  "federated_identity_services": null,
+  "federated_identity_services": ["-invalid.host.com"],
```

Comment:
Why this change ? null means no fd service

Reply:
i made this change to have a better test coverage - invalid host names are filtered out from the config when creating the MatrixIdentityServer

Reply:
Not the good file. Here is the default config